docs/guide/javaee/jpa.adoc

```diff
@@ -1,11 +1,11 @@
 ### JPA
 
 JPA integration is provided by the http://camel.apache.org/jpa.html[Camel JPA component,window=_blank].
-Camel JPA applications are developed as per normal by providing a persistence.xml configuration file together with some JPA annotated classes.
+You can develop the Camel JPA applications by providing a persistence.xml configuration file together with some JPA annotated classes.
```

Comment:
Change 'You can develop the Camel JPA applications' to 'You can develop Camel JPA applications'.

Reply:
Fixed, as suggested.